.github/workflows/deploy_tre.yml

```diff
@@ -108,3 +108,4 @@ jobs:
       TF_STATE_CONTAINER: ${{ secrets.TF_STATE_CONTAINER }}
       TRE_ADDRESS_SPACE: ${{ secrets.TRE_ADDRESS_SPACE }}
       TRE_ID: ${{ format('tre{0}', needs.prepare-not-main.outputs.refid) }}
+      SECONDARY_CACHE_ACR_NAME: $${{ secrets.ACR_NAME }}
```

Comment:
Is this the same value as line 87? I'm assuming that this is the tre_acr that main uses? If that's the case, would it be safer to use ACTIONS_ACR_NAME? Just thinking out loud that someone could write some Terraform that dropped the repo

Reply:
No it's not the save value as in 87. 
@marrobi and I had a discussion before this whole env in pr validation started - it would have been ideal if we could use the same ACR across all environments, but there're challenges there (especially because of porter).
If we use the ACTIONS_ACR then we'll have to push those images there too which we currently don't.
I think this is a good compromise that will only reference the main ACR as a cache but wouldn't interact with it which is something the run from main is responsible of doing.
I think that "cache" wise it will be virtually be the same since the previous version of an image in the new branch is likely be the latest from main (which is where the new branch came from).

Reply:
I'm happy with that - just going to be hard for someone in a year to work out all the different acr's! But yes I agree with the compromise